ntbs-service/Models/NtbsContext.cs

```diff
@@ -126,5 +144,38 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 new Sex { SexId = 3, Label = "Unknown" }
             );
         }
+
+        private List<Object> GetHospitalsList()
```

Comment:
This method didn't play nicely when having it generic and passing anonymousTypeDefinition as parameter.
And now thinking about it, I should probably move this to its own helper class that extends SeedingHelper class

Reply:
Yeah, seems sensible to crack this out separate. No worries about the generic atm I think, keep it simple for now and if more of these show up then we can see where that steers us!